### PR TITLE
Flash preload address code review feedback

### PR DESCRIPTION
**Pull request by pajong**
Only preload when "metadata" or "auto", change var name, remove _preload tag, remove unnecessary call to loadStream in seek

### CODE REVIEW

File: src/flash/com/longtailvideo/jwplayer/media/VideoMediaProvider.as [hunk 1]
```diff
@@ -88,47 +88,29 @@ public class VideoMediaProvider extends MediaProvider {
...
-        // Set initial quality and set levels
-        _currentQuality = 0;
-
-        for (var i:Number = 0; i < _item.levels.length; i++) {
-            if (_item.levels[i]["default"]) {
-                _currentQuality = i;
-                break;
-            }
+    override public function init(itm:PlaylistItem):void {
+        if (itm.preload !== "metadata" && itm.preload !== "auto") {
```
Comment (donato) [inline src/flash/com/longtailvideo/jwplayer/media/VideoMediaProvider.as]: Check here

File: src/js/providers/flash.js [hunk 2]
```diff
@@ -103,7 +103,25 @@ define([
...
+                    if (item.preload === 'none' || _playerConfig.autostart) {
+                        return;
+                    } else {
+                        _item = item;
+
+                        // if we just call flashCommand with _swf undefined, init will go into commandQueue
+                        if (_swf) {
+                            _flashCommand('init', item);
+                        } else {
+                            this.once('swfDefined', function() {
```
Comment (donato) [inline src/js/providers/flash.js]: try to reproduce this case, I'm not sure it's possible

File: src/flash/com/longtailvideo/jwplayer/player/Player.as [hunk 3]
```diff
@@ -26,6 +26,8 @@ public class Player extends Sprite implements IPlayer {
 
     protected var _instream:InstreamPlayer;
 
+    private var _preloaded:Boolean;
```
Comment (donato) [inline src/flash/com/longtailvideo/jwplayer/player/Player.as]: Test without using this, relying on the controllers file check

File: src/js/providers/flash.js (hunk 2 above)
Comment (pajong) [inline src/js/providers/flash.js]: This actually happens, and init goes into "queueCommand", which is ran when we are trying to load.

**Pull request closed without merging**